libc/src/math/generic/CMakeLists.txt

```
@@ -129,6 +129,58 @@ add_entrypoint_object(
     libc.src.__support.FPUtil.nearest_integer_operations
 )
 
+add_entrypoint_object(
+  daddl
+  SRCS
+    daddl.cpp
+  HDRS
+    ../daddl.h
+  COMPILE_OPTIONS
+    -O3
+  DEPENDS
+    libc.src.__support.macros.properties.types
+    libc.src.__support.FPUtil.generic.add_sub
```

`daddl` doesn't use `float128`.

```suggestion
  DEPENDS
    libc.src.__support.FPUtil.generic.add_sub
```